.ci/trial-json-schema.json

```diff
@@ -58,6 +58,12 @@
       "type" : "object",
       "additionalProperties" : false,
       "properties" : {
+        "profile" : {
+          "type" : "string"
+        },
+        "label" : {
+          "type" : "string"
+        },
```

Comment:
1. Are those the right names to indicate what they will be used for?
2. Their location in the root of the spring_cloud element also indicates that they will be used across spring cloud services. Will they?

Reply:
as discussed f2f, moved to upper level. 
profile is now spring_profile as its affects both the properties of each service + affect which profile will be reach in the config server
label is now spring_config_label as it affects ONLY the branch of the config server